matchbox_socket/src/webrtc_socket/socket.rs

```diff
@@ -430,6 +443,19 @@ impl WebRtcSocket {
 }
 
 impl<C: ChannelPlurality> WebRtcSocket<C> {
+    /// Disconnect from the peer, severing all communication channels.
+    pub fn disconnect(&mut self, peer: PeerId) {
+        todo!("Needs attention from @johanhelsing or @garryod")
```

Comment:
Ok, hello! Once again calling for some assistance @johanhelsing and @garryod. I was able to implement close() quite well, however, I'm at a serious brick wall in implementing WebRtcSocket::disconnect(peer: PeerId)`. I added the todo here, please have a stab at it!